### PR TITLE
Add button component page

### PR DESCRIPTION
**Pull request by Yanhuazong**

### CODE REVIEW

File: src/html/components/mm/templates/content-accordion.handlebars [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{> header}}
-<main id='contentTemplate'> 
+<main id='accordionTemplate'>
```
Comment (jyoung295) [inline src/html/components/mm/templates/content-accordion.handlebars]: Should this be `buttonTemplate` ?

**Comment by jyoung295** [comment]
There's an issue with the slim buttons on desktop:

[Image from Gyazo](https://gyazo.com/754724d0241916e73d8a6a469da3bd0c)

There's extra spacing at the bottom

File: src/html/components/mm/templates/content-accordion.handlebars (hunk 1 above)
Comment (Yanhuazong) [reply src/html/components/mm/templates/content-accordion.handlebars]: I'm not sure. Do you mean it should be contentTemplate?

**Comment by Yanhuazong** [comment]
Fixed in the new commit. Thanks!

File: src/html/components/mm/templates/content-accordion.handlebars (hunk 1 above)
Comment (jyoung295) [reply src/html/components/mm/templates/content-accordion.handlebars]: Well I saw that you had `accordionTemplate` on the accordion example page you made, so following that idea, then this should be `buttonTemplate` or we could make all of the example pages also just use `contentTemplate`

File: html-render.js [hunk 2]
```diff
@@ -57,56 +57,52 @@ const registerButtonSectionHelper = () => {
...
-    const headerH33 = '</div><h3>Light, Solid</h3><h4>Button Elements</h4><div class="button-group">';
-    const headerH34 = '</div><h3>Light, Hollow</h3><h4>Button Elements</h4><div class="button-group">';
-    const headerH35 = '</div><h3>Light, Bar</h3><h4>Button Elements</h4><div class="button-group">';
-    const headerH36 = '</div><h3>Light, Opaque</h3><h4>Button Element</h4><div class="button-group" style="background-image:url(\'https://placeimg.com/767/300/nature\'); background-size:cover; background-position:top center;">';
-    const headerH37 = '</div><h3>Default</h3><div class="button-group">'
-    const headerH41 = '</div><h4>Button Elements</h4><div class="button-group">';
-    const headerH42 = '</div><h4>Anchor Elements</h4><div class="button-group">';
-    const headerH422 = '</div><h4>Anchor Element</h4><div class="button-group" style="background-image:url(\'https://placeimg.com/767/300/nature\'); background-size:cover; background-position:top center;">';
+    const headerH31 = '</div><h3>Anchor Elements</h3><div class="button-group">';
```
Comment (jyoung295) [inline html-render.js]: There's probably a better way to name these variables to be more readable in the future if we ever return to this.

**Pull request merged**